modules/tinymce/src/core/test/ts/browser/fmt/PreviewTest.ts

```diff
@@ -1,238 +1,236 @@
-import { Assertions, Pipeline, Step, Waiter } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+import { Assertions, Logger, Pipeline, Step, Waiter } from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
 import { Arr } from '@ephox/katamari';
-import { LegacyUnit, TinyLoader } from '@ephox/mcagar';
-import Editor from 'tinymce/core/api/Editor';
+import { TinyLoader } from '@ephox/mcagar';
 import * as Preview from 'tinymce/core/fmt/Preview';
 import Theme from 'tinymce/themes/silver/Theme';
 import * as HtmlUtils from '../../module/test/HtmlUtils';
 
-UnitTest.asynctest('browser.tinymce.core.fmt.PreviewTest', function (success, failure) {
-  const suite = LegacyUnit.createSuite<Editor>();
+UnitTest.test('Preview.parseSelector()', () => {
```

Comment:
Another one where the prefix in the name shouldn't have been removed

Reply:
Fixed.